README.md

```diff
@@ -31,6 +31,16 @@ end
 
 See [opentracing-ruby](https://github.com/opentracing/opentracing-ruby) for more examples.
 
+### Samplers
+
+#### Const sampler
+
+`Const` sampler always makes the same decision for new traces depending on the initialization value. Set `sampler` to: `Zipkin::Samplers::Const.new(true)` to mark all new traces as sampled.
+
+#### Probabilistic sampler
+
+`Probabilistic` sampler samples traces with probability equal to `rate` (must be between 0.0 and 1.0). Set `sampler` to `Zipkin::Samplers::Probabilistic.new(rate: 0.1)` to mark 10% of new traces as sampled.
```

Comment:
Isn't `probabilistic` sampler with sample rate eq `1.0` the same as `const` sampler? What do you think of having just a `sample_rate` configuration option?

Reply:
It is same. I saw it implemented like this in multiple places: e.g. https://github.com/jaegertracing/jaeger-client-go#sampling . Maybe it's just one calculation less, although it should be really fast.

Reply:
```
def sample?(*)
  return true if @rate == 1.0

  ...
end
```

this would cover this optimization

Reply:
I'm fine with current implementation though, just thought that it'd be easier to understand and set up